thunder/torch/__init__.py

```diff
@@ -2164,6 +2164,16 @@ def addcdiv_(a: TensorLike, b: TensorLike, c: TensorLike, /, *, value: None | Nu
     return prims.copy_(addcdiv(a, b, c, value=value), a)
 
 
+@torchsymbol(torch.lerp, is_method=True)
+def lerp(start: TensorLike, end: TensorLike, weight: Number | TensorLike) -> TensorLike:
+    return clang.lerp(start, end, weight)
+
+
+@torchsymbol(torch.Tensor.lerp_, is_method=True, tags=(prims.OpTags.IN_PLACE,))
+def lerp_(start: TensorLike, end: TensorLike, weight: Number | TensorLike) -> TensorLike:
+    return prims.copy_(lerp(start, end, weight), start)
```

Comment:
fyi @crcpar

When doing `copy_` we might want to require the arguments be specified as keywords, or following PyTorch's convention that the first tensor is the one that's modified.

For example

https://github.com/Lightning-AI/lightning-thunder/blob/ceba7307d1ec6c43683a5f5bcf376dfb1841d240/thunder/core/utils.py#L169

requires the kwarg names be used so there's never confusion about what is being copied from or copied to